R/class-plotdata-map-markers.R

```diff
@@ -64,8 +85,12 @@ newMapMarkersPD <- function(.dt = data.table::data.table(),
       if (length(rankedValues) > 8) {
         rankedValues <- c(rankedValues[1:7], 'Other')
         .pd[[x]][!.pd[[x]] %in% rankedValues] <- 'Other'
+        overlayValues <- c(stringi::stri_sort(rankedValues[1:7], numeric=TRUE), 'Other')
+      } else {
+        overlayValues <- stringi::stri_sort(rankedValues[1:7], numeric=TRUE)
```

Comment:
Perhaps just `stringi::stri_sort(rankedValues, numeric=TRUE)` here? We know that the length of rankedValues is 8 or less, so we should be able to just sort the vector i'd think?